Lib/test/test_capi/test_import.py

```diff
@@ -317,6 +318,44 @@ def test_executecodemoduleobject(self):
         # CRASHES execute_code_func(NULL, code, NULL, NULL)
         # CRASHES execute_code_func(name, NULL, NULL, NULL)
 
+    def check_getmoduleattr(self, getmoduleattr):
+        self.assertIs(getmoduleattr('sys', 'argv'), sys.argv)
```

Comment:
Check also a module name containing dots.

Check errors:
* non-existent module
* relative module name
* non-existent attribute
* attribute name containing dots

Reply:
I added requested tests.

> attribute name containing dots

`getmoduleattr('sys', 'implementation.name')` raises an `AttributeError`. I prefer to not implement "nested" `getattr()`. One should write `getmoduleattr('sys', 'implementation')` + `PyObject_GetAttrString(attr, "name")`.